Remotion/Web/ClientScript/Scripts/DropDownMenu.ts

```diff
@@ -533,14 +533,11 @@ class DropDownMenu
     popUpDiv.style.position = 'fixed';
 
     // move dropdown if there is not enough space to fit it on the page
-    if ((LayoutUtility.GetWidth (popUpDiv) > space_left) && (space_left < space_right))
+    if ((LayoutUtility.GetWidth (popUpDiv) > (space_left + LayoutUtility.GetWidth(referenceElement))) && (space_left < space_right))
     {
-      if (LayoutUtility.GetOffset (popUpDiv).left < 0)
-      {
-        left = Math.max (0, LayoutUtility.GetOffset (referenceElement).left) + 'px';
-        popUpDiv.style.left = left;
-        popUpDiv.style.right = 'auto';
-      }
+      left = Math.max (0, LayoutUtility.GetOffset (referenceElement).left) + 'px';
+      popUpDiv.style.left = left;
+      popUpDiv.style.right = 'auto';
     }
```

Comment:
It seems that not everything is solved:
![image](https://user-images.githubusercontent.com/6972548/163426694-d4ae7429-e122-47bf-812c-2731cae7f6da.png)
There's also no input regarding the use of an elispis with overflow and if using a left/right-margin instead of sticking to the window border is feasible.